Disable online updater as libretro cores are built for glibc.
--- a/retroarch.cfg
+++ b/retroarch.cfg
@@ -702,7 +702,7 @@
 # menu_show_online_updater = true
 
 # If disabled, will hide the ability to update cores (and core info files) inside the menu.
-# menu_show_core_updater = true
+menu_show_core_updater = false
 
 # If disabled, the libretro core will keep running in the background when we
 # are in the menu.
@@ -831,7 +831,7 @@
 
 # Assets directory. This location is queried by default when menu interfaces try to look for
 # loadable assets, etc.
-# assets_directory =
+assets_directory = /usr/share/libretro/assets
 
 # Dynamic wallpapers directory. The place to store the wallpapers dynamically
 # loaded by the menu depending on context.
@@ -844,16 +844,16 @@
 # rgui_browser_directory =
 
 # Core directory for libretro core implementations.
-# libretro_directory =
+libretro_directory = /usr/lib/libretro
 
 # Core info directory for libretro core information.
-# libretro_info_path =
+libretro_info_path = /usr/share/libretro/info
 
 # Path to content database directory.
 # content_database_path =
 
 # Path to cheat database directory.
-# cheat_database_path =
+cheat_database_path = /usr/share/libretro/database/cht
 
 # Defines a directory where CPU-based video filters are kept.
 # video_filter_dir =
@@ -882,7 +882,7 @@
 # Input binds which are made explicit (input_playerN_*_btn/axis) will take priority over autoconfigs.
 # Autoconfigs can be created with manually, or with the frontend.
 # Requires input_autodetect_enable to be enabled.
-# joypad_autoconfig_dir =
+joypad_autoconfig_dir = /usr/share/libretro/autoconfig
 
 # Save all remapped controls to this directory.
 # input_remapping_directory =
